src/models/YotiPayloads.ts

```diff
@@ -43,6 +43,13 @@ export interface CreateSessionPayload {
 			scheme?: string;
 		};
 	}>;
+	requested_tasks: Array<{
```

Comment:
Does this require modifying the Yoti spec at https://github.com/alphagov/di-ipv-cri-f2f-api/blob/main/yoti-stub/yoti-spec.yaml at all?

Reply:
Pretty sure it's already there: https://github.com/alphagov/di-ipv-cri-f2f-api/blob/main/yoti-stub/yoti-spec.yaml#L299